packages/sanity/src/tasks/src/tasks/components/activity/TasksActivityLog.tsx

```diff
@@ -116,9 +118,11 @@ export function TasksActivityLog(props: TasksActivityLogProps) {
 
       onChange(set(notification.subscribers, ['subscribers']))
 
-      operation.create(nextComment)
+      operation.create(nextComment).then(() => {
+        telemetry.log(TaskCommentAdded)
```

Comment:
Nice!

Suggestion/idea: This is not in scope for this pull request, but maybe we should consider (in the future) to centralize the logging of comment-related telemetry events within the `useCommentOperations` hook, rather than in the places where those operations are invoked? Currently, we do not log telemetry for comments, so we could consider this as a refactoring effort in the future when we implement telemetry for comments.

Something like this (pseudo code):

```ts
import {
  TaskCommentCreated,
  FieldCommentCreated,
  TaskReplyCommentCreated,
  FieldReplyCommentCreated,
} from '../path/to/telemetry'
import {useTelemetry} from '@sanity/telemetry'

function useCommentOperations(props: CommentOperationsHookOptions) {
  const {type} = props
  const telemetry = useTelemetry()

  const handleCreate = useCallback(
    async (nextComment) => {
      await createOperation(nextComment).then(() => {
        const isReply = Boolean(nextComment.parentCommentId)

        if (!isReply) {
          if (type === 'task') {
            telemetry.log(TaskCommentCreated)
          }

          if (type === 'field') {
            telemetry.log(FieldCommentCreated)
          }
        }

        if (isReply) {
          if (type === 'task') {
            telemetry.log(TaskReplyCommentCreated)
          }

          if (type === 'field') {
            telemetry.log(FieldReplyCommentCreated)
          }
        }
      })
    },
    [telemetry, type],
  )

  return {
    create: handleCreate,
    // ... rest of the operations
  }
}
```

Reply:
I'll create a issue to track that, I think we should be tracking similar events for comments specific changes, but let's raise it with the growth team

Reply:
I believe comments are tracked server-side? We do have events for Comment Posted etc in Amplitude

Reply:
That's right, actually I will remove this comments events from the studio and move them to the server, I can add a `type` property to distinguish normal comment from tasks comments.